apps/native-component-list/src/screens/Audio/expo-audio/Recorder.tsx

```diff
@@ -44,32 +49,25 @@ export default function Recorder({ onDone, style }: RecorderProps) {
     })();
   }, []);
 
-  const audioRecorder = useAudioRecorder(
-    {
-      extension: '.mp4',
-      android: {
-        outputFormat: 'mpeg4',
-        audioEncoder: 'default',
-      },
-      ios: {
-        linearPCMBitDepth: 16,
-        linearPCMIsBigEndian: false,
-        linearPCMIsFloat: false,
-        audioQuality: AudioQuality.MAX,
-      },
-      sampleRate: 44100,
-      numberOfChannels: 2,
-      bitRate: 128000,
-    },
-    (status) => {
-      setState(status);
-    }
-  );
+  const audioRecorder = useAudioRecorder(recorderOptions, (status) => {
+    setState(status);
+  });
 
   const recorderState = useAudioRecorderState(audioRecorder);
 
   const record = () => audioRecorder.record();
 
+  const prepare = async () => {
+    await audioRecorder.prepareToRecordAsync();
+  };
```

Comment:
Do we need those? 😅 Can't we just call them directly?

Reply:
Could inline it but passing a reference to a function on a shared object does not work correctly